data/mappings/info_config.json

```diff
@@ -38,5 +38,9 @@
     "RGBLIGHT_SPLIT": {"info_key": "rgblight.split", "value_type": "bool"},
     "PRODUCT": {"info_key": "keyboard_folder", "to_json": false},
     "PRODUCT_ID": {"info_key": "usb.pid", "value_type": "hex"},
-    "VENDOR_ID": {"info_key": "usb.vid", "value_type": "hex"}
+    "VENDOR_ID": {"info_key": "usb.vid", "value_type": "hex"},
+    "QMK_ESC_OUTPUT": {"info_key": "qmk_dfu.esc_output"},
```

Comment:
Thinking this should be less tied to DFU, as these defines could also be used for the LUFA HID bootloader.

Reply:
I originally had "bootloader" but that was already taken, will have a think as i also didnt like how it was "qmk-dfu" in one context but then "qmk_dfu" elsewhere.

Reply:
Maybe `qmk_lufa_bootloader`? That could also give a hint as to the fact QMK-DFU (and potentially QMK-HID) are only for LUFA-based boards.